src/main/java/com/wepay/model/data/AddressData.java

```diff
@@ -26,4 +26,15 @@ public static JSONObject buildAddress(AddressData info) throws JSONException {
 		return o;
 	}
 
+	public static JSONObject buildUnifiedAddress(AddressData info) throws JSONException {
```

Comment:
Not yet. The unified address format will become the standard in the next version of the SDK. The version of the WePay API we're using here still uses the older address format, except in one case.